### PR TITLE
pgcli 0.17.0

### PR DESCRIPTION
**Pull request by amjith**
Upgraded the formula for the latest version of pgcli. (http://pgcli.com).

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Install is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/26764/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_pgcli/

**Comment by amjith** [comment]
Fixed the failure and update the PR. Waiting for it to finish the jenkins build.

File: Library/Formula/pgcli.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Pgcli < Formula
   homepage "http://pgcli.com/"
-  url "https://pypi.python.org/packages/source/p/pgcli/pgcli-0.16.3.tar.gz"
-  sha1 "747e8505514255833738a0ded3809728d942dda7"
+    url "https://pypi.python.org/packages/source/p/pgcli/pgcli-0.17.0.tar.gz"
```
Comment (DomT4) [inline Library/Formula/pgcli.rb]: Can you fix the indent here? It needs to be lined up under `homepage`.

**Comment by amjith** [comment]
Indentation is fixed.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/1bcaecee52429e543c9ba1f8d5c77cc06b23f35f. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:.

Please note the preferred commit message for simple version bumps is `pgcli 0.17.0` - You got it right in the PR title :). Ideally commits should be squashed as well. Otherwise absolutely perfect, Thanks!

**Pull request closed without merging**